src/abstractdataframe/abstractdataframe.jl

```diff
@@ -405,31 +410,100 @@ describe(df)
 ```
 
 """
-StatsBase.describe(df::AbstractDataFrame) = describe(stdout, df)
-function StatsBase.describe(io, df::AbstractDataFrame)
-    for (name, col) in eachcol(df)
-        println(io, name)
-        describe(io, col)
-        println(io, )
+StatsBase.describe(df::AbstractDataFrame; kwargs...) = describe(stdout, df; kwargs...)
+function StatsBase.describe(io, df::AbstractDataFrame; stats = [:mean, :min, :median, :max, :nmissing, :eltype])
+     # Check that people don't specify the wrong fields. 
+    allowed_fields = [:mean, :std, :min, :q25, :median, :q75, :max, :eltype, :nunique,:nmissing] 
+    if !issubset(stats, allowed_fields) 
+        disallowed_fields = setdiff(stats, allowed_fields)
+       error("Not an allowed field. Allowed fields are:mean, :min, :q25, 
```

Comment:
Added the `throw(ArgumentError(...))` but I kept the list of allowed fields for the user to see what they probably mis-typed (`sd` instead of `std`, for example).